client/src/app/pages/login/login.component.html

```diff
@@ -1 +1,16 @@
-<p>login works!</p>
+<div class="title">
+    <h1>Login</h1>
+</div>
+<div class="logo">
+
+        <img src="../../../assets/images/patitas.png" alt="">
+
+</div>
+<div class="form">
+    <input type="text" placeholder="Email">
+    <input type="password" placeholder="Password">
+    <i class="fa-sharp fa-solid fa-eye-slash"></i>
+    <p>¿Olvidaste tu Contraseña? <a href="#">Recuperar</a></p>
+    <button>Iniciar Sesion</button>
+    <p [routerLink]="">¿No tienes una cuenta? <a href="#">Registrate</a></p>
```

Comment:
I recommend you to change this line to `<p>¿No tienes una cuenta? <a routerLink="/register">Registrate</a></p>` to respect the SPA concept.

Reply:
Im agree! i will change